packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -738,7 +743,7 @@ function createInstances(cluster: DatabaseClusterNew, props: DatabaseClusterBase
       dbParameterGroupName: instanceParameterGroupConfig?.parameterGroupName,
       monitoringInterval: props.monitoringInterval && props.monitoringInterval.toSeconds(),
       monitoringRoleArn: monitoringRole && monitoringRole.roleArn,
-      autoMinorVersionUpgrade: props.instanceProps.autoMinorVersionUpgrade,
+      autoMinorVersionUpgrade: autoMinorVersionUpgrade,
```

Comment:
Let's get rid of the `autoMinorVersionUpgrade` variable, and just do things inline:

```suggestion
      autoMinorVersionUpgrade: props.engine.engineVersion?.fullVersion && props.instanceProps.autoMinorVersionUpgrade === undefined
        // if engine version is pinned and autoMinorVersionUpgrade is not passed in props, we set this property to false
        ? false
        : props.instanceProps.autoMinorVersionUpgrade,
```

Reply:
Done.thanks.